test/FluentMigrator.Tests/Integration/Migrations/TestMigration.cs

```diff
@@ -30,16 +30,16 @@ public override void Up()
             // successfully tested. At time of implementing, the FK constraint
             // wasn't explicitly used by any tests and so should affect anything.
 
+            Create.Table("Groups")
+                .WithColumn("GroupId").AsInt32().Identity().PrimaryKey()
+                .WithColumn("Name").AsString(32).NotNullable();
+
```

Comment:
Yes it failed before this change : 
![image](https://github.com/user-attachments/assets/28bc4fe4-175e-4893-9bc0-e9910d8d5aa5)

And other tests after the first error fail too, because the Users table already exists, I guess because something went wrong during the cleanup.